types/angular-aria/tslint.json

```diff
@@ -0,0 +1,6 @@
+{
+    "extends": "dtslint/dt.json",
+    "rules": {
+        "interface-name": false
```

Comment:
So the current DT style guide suggests to not put the exclude rule here in the config file, but explicitely use this (if possible) in the source code, so for example:
```ts
/**
* see {@link https://docs.angularjs.org/api/ngAria/provider/$ariaProvider}
*/
// tslint:disable-next-line interface-name
interface IAriaOptions {
```

Reply:
Yeah but all AngularJS typings break this rule. Which is why I would rather disable it package wide.